e2e/wrappers/test-helpers.js

```diff
@@ -0,0 +1,21 @@
+import { fixture, test } from 'testcafe';
+const process = require('process');
+
+export function navigateToComponent(t, baseUrl, examplePath) {
+    return t.navigateTo(`${baseUrl}/examples/${examplePath}`);
+}
+
+export function testInFramework(examplePath, testFn) {
+    const frameworkName = process.env.FRAMEWORK || 'react';
+    const port = process.env.E2E_TEST_PORT || 3030;
+    const baseUrl = `http://localhost:${port}`;
+
+    fixture(`${frameworkName.toUpperCase()} Framework`)
```

Comment:
By looking at the `button.test.js` file, I can assume the `testInFramework` function is called for each test.

But here I see that it creates a fixture that is named like it's the only one for each framework.

I am afraid I don't completely understand the hierarchy of entities here...